src/SelectMulti.vue

```diff
@@ -215,17 +215,21 @@
 		</label>
 		<ul :id="`${htmlId}-selected`" class="selected-options" :class="{ 'below-input': displayPillsBelowInput }">
 			<span :id="`${htmlId}-selected-option-pills`" style="display: none;">remove</span>
-			<li v-for="(option, index) in selectedOptions" :key="option.value">
-				<button
-					class="selected-option-pill"
-					:disabled="disabled"
-					type="button"
-					:aria-describedby="`${htmlId}-selected-option-pills`"
-					@click="removeOption(index)"
-				>
-					{{ option.label }}
-				</button>
-			</li>
+			<template v-for="(option, index) in selectedOptions" >
+				<li v-if="option.value" :key="option.value" >
+					<button
+						class="selected-option-pill"
+						:disabled="disabled"
+						type="button"
+						:aria-describedby="`${htmlId}-selected-option-pills`"
+						@click="removeOption(index)"
+					>
+						<slot name="selectedOption" v-bind:option="option">
```

Comment:
Can we omit `v-bind` and leave only `:option="option"`?

Reply:
Yes, good catch